### ISSUE TITLE
Download Button on Interactive GRaaS Report Doesn't Work on Safari Browser

### ISSUE THREAD

**Issue by aliatt2** [cal-itp/graas#397]
The interactive GRaaS report allows us to download a daily report for agencies running on GRaaS. The download button does not work on Safari browser. I've confirmed that it works as expected on Chrome, but when I click it on Safari, nothing happens. 

<img width="1437" alt="image" src="https://user-images.githubusercontent.com/92057994/223235074-8df970fe-41cc-4727-99bc-2c391c4ed660.png">

**Comment by unreasonableman** [comment]
Safari is the only major browser to have attitude about support for a javascript feature called `OffScreenCanvas`. We are using this feature for downloads. See table below for the state of support affairs.

With that said, safari implements basic support for this feature starting at version 16.4, which is enough for us to get working downloads. **Heads-up**: I had to update to MacOS Monterey 12.6.4 to be able to get a safari version >= 16.4.

In summary, I was able to reproduce this issue before updating safari, and have confirmed that the issue is fixed after updating.

<img width="789" alt="offscreen-canvas-support" src="https://user-images.githubusercontent.com/833549/224841845-fe86b17e-4987-4aa6-9b22-c6f039f7741c.png">
